package.xml

```diff
@@ -8,8 +8,14 @@
   <license>Apache2</license>
 
   <buildtool_depend>ament_cmake_auto</buildtool_depend>
-  <exec_depend>xacro</exec_depend>
-  <!-- <exec_depend>pacmod3</exec_depend> --> <!-- not released for the Foxy yet -->
+
+  <depend>xacro</depend>
```

Comment:
Why has this become `depend` from `exec_depend`?

Reply:
I fixed it in https://github.com/tier4/lexus_description.iv/pull/24/commits/0f263806774284fb1dd6615f8a69aec47f14ee50